database/sqlite/sqlite_aclk_chart.c

```diff
@@ -944,6 +941,24 @@ void aclk_update_retention(struct aclk_database_worker_config *wc, struct aclk_d
 
         if (likely(!rc && first_entry_t))
             start_time = MIN(start_time, first_entry_t);
+
+        if (wc->chart_updates) {
+            int live = ((now - last_entry_t) < (RRDSET_MINIMUM_DIM_LIVE_MULTIPLIER * update_every));
+            if ((!live || !first_entry_t) && (dimension_update_count < ACLK_MAX_DIMENSION_CLEANUP)) {
+                (void)aclk_upd_dimension_event(
```

Comment:
Hi Stelio. This passes return values from the sql query, but in case of memory mode not dbengine, it will execute SELECT_HOST_CHART_LIST which does not appear to return the same columns as SELECT_HOST_DIMENSION_LIST ?

Reply:
I will push a fix since this will only work for dbengine